src/app/components/cds/charts/NumberWidget.js

```diff
@@ -13,12 +13,21 @@ const NumberWidget = ({
     <div className="typography-subtitle2">
```

Comment:
oh also @vasconsaurus unrelated to this PR, but I noticed that in the js file here, you are using the typography class to set the styles, but in the css file for this, you also have `.numberWidgetTitle` which is unused. So you probably either want to use that className or remove it from the css file

Reply:
Missed this! Thank you!